src/Utils/typeSupport.ts

```diff
@@ -0,0 +1,35 @@
+/** Remove the special relay properties from an interface */
+export type DeFraged<T> = Omit<T, " $refType" | " $fragmentRefs">
```

Comment:
I was thinking we needed a centralized way to do this. I was manually doing this https://github.com/artsy/reaction/blob/7b712fc8390f0aca42f3ac53f292a4a6a753db20/src/Apps/Artwork/Components/TrustSignals/__tests__/SecurePayment.test.tsx#L11

It might be worth doing a search for `" $refType"` and updating any tests/examples that use this. (That could be a follow up)